packages/product-components/src/components/PassphraseTypeCard/getSubmitLabel.tsx

```diff
@@ -0,0 +1,28 @@
+import type { ReactNode } from 'react';
+import React from 'react';
+import { Translation } from '../../../../suite/src/components/suite/Translation';
```

Comment:
I don't like this solution, as there is not a single usage of `Translation` in `/packages/product-components` so far, so it seems I'm breaking some kind of rule here. 

Also, I know that I'm creating circular dependency with this. It seems that the ideal solution for me would be to extract Translation from `suite` so it could be used even here 🤔

The other option that I considered was to pass the translations from the above level, but then more refactoring would be needed as we don't have access to the `value` on the above level. Also it does not make sense to pass it from the level above as I believe these Translations will never change and should be part of the component 🤔 I'm interested in your thoughts on this topic 🙏

Reply:
We use 
`<FormattedMessage id="ID_HERE" />`